Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -141,6 +143,7 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
     }
 
     override func deleteBackward() {
+        canAutoComplete = false
         removeCompletion()
         super.deleteBackward()
         autocompleteDelegate?.autocompleteTextField(self, didEnterText: self.text)
```

Comment:
Can we also set `enteredTextLength = count(self.text)` here? We should try to keep it in sync whenever the text changes.